.github/workflows/versioned-security-agent.yml

```diff
@@ -39,34 +39,25 @@ jobs:
     name: Should Run
     runs-on: ubuntu-latest
     outputs:
-      previous_version: ${{steps.versions.outputs.PREVIOUS_VAL}}
-      current_version: ${{steps.versions.outputs.CURRENT_VAL}}
+      sec_agent_did_change: ${{ steps.check_sec_agent.outputs.divergent }}
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 2
-      - uses: tj-actions/changed-files@v44
-        id: changed_files
-        with:
-          files: package.json
-      - name: Get dependency versions
-        id: versions
+      - name: Check Security Agent Changed
+        id: check_sec_agent
         if: steps.changed_files.outputs.any_changed == 'true'
```

Comment:
i know this wasn't changed but where is this job?

Reply:
I removed it but forgot to remove this line. Good catch.